ecl/hqlcpp/hqlres.cpp

```diff
@@ -648,7 +648,8 @@ bool ResourceManager::flush(StringBuffer &filename, const char *basename, bool f
         }
         fwrite(s.data.get(), 1, s.data.length(), bin);
         fclose(bin);
-        fprintf(f, " .size %s,%u\n", label.str(), (unsigned)s.data.length());
+        if (!generateClang)
```

Comment:
this looks like an unrelated bug.  If so it will need to target an earlier build.

Reply:
I don't know if it's in general a change that is needed - but I needed it to be able to compile ANY workunit on my current mac system